packages/participants-svc/src/domain/entities/participant.ts

```diff
@@ -173,14 +176,33 @@ export declare class ParticipantEndpoint implements IParticipantEndpoint{
 }
 
 
-export declare class ParticipantAccount implements IParticipantAccount{
+export declare class ParticipantAccounts implements IParticipantAccounts{
```

Comment:
Same as IParticipantAccounts -> This should be kept as singular (it stores/holds a single entity).